### ISSUE TITLE
Configuration issue

### ISSUE THREAD

**Issue by 123dev** [Graylog2/graylog-plugin-aws#16]
How does one configure the latest version of this plugin?
The documentation states the following.

> After installing the plugin you will have a new cluster configuration section at “System -> Configurations” in your Graylog Web Interface.

Yet navigating to the System -> Configurations show no options for this plugin (see attached image)
![2016-09-08 11_33_44-graylog web interface](https://cloud.githubusercontent.com/assets/13109154/18355856/822fbb2c-75b8-11e6-8db0-1a8f0dca5a1c.png)

The plugin is properly installed, the logs show

`2016-09-08 15:33:59,208 WARN : org.graylog.aws.processors.instancelookup.AWSInstanceNameLookupProcessor - AWS plugin is not fully configured. No instance lookups will happen.
2016-09-08 15:33:59,723 WARN : org.graylog.aws.processors.instancelookup.AWSInstanceNameLookupProcessor - AWS plugin is not fully configured. No instance lookups will happen.
2016-09-08 15:33:59,887 WARN : org.graylog.aws.processors.instancelookup.AWSInstanceNameLookupProcessor - AWS plugin is not fully configured. No instance lookups will happen.
`

I can see the plugin in the Graylog Inputs
Which offers no options for configuring credentials or other settings, it only asks for Region.
(see image)
![2016-09-08 11_18_25-graylog web interface](https://cloud.githubusercontent.com/assets/13109154/18355937/d19fefa6-75b8-11e6-8582-3b72852c6610.png)

Obviously the input can't be started without the proper configuration.

What am I missing?
Thanks

**Comment by 123dev** [comment]
Just to add, I'm on the latest version of Graylog
2.1.0 as of this writing.

**Comment by brucegivens** [comment]
Having the same issue here, Graylog 2.1.1 and AWS Plugin 1.0.0.

**Comment by 123dev** [comment]
Bruce,
A fundamental issue such as this, has fallen on deaf ears.
This being free and open source, we can't be demanding, hence we've given up hope getting the author's attention.

Would appreciate if you post here a follow up, if you ever figure out a solution.
It's a pity, because it was great plugin when it used to work.

**Comment by 123dev** [comment]
Update:
I just upgraded to Graylog 2.1.1 and the now I can see the configuration options.
Although this is solved for us, @brucegivens has reported that he has a problem with Graylog 2.1.1
and considering that we also had a problem with Graylog 2.1.0, I'll say there's definitely a problem that warrants this ticket to be kept open.

![graylog_2_1_1](https://cloud.githubusercontent.com/assets/13109154/19402973/450b7a60-9232-11e6-8864-3a6929c7f2de.png)

**Comment by brucegivens** [comment]
I've downloaded the plugin again and restarted graylog and am now also seeing the plugin configuration section. To the best of my knowledge, no changes have been made to server.conf in the meantime - I simply tried again and it worked this time.
@123dev: did you restart graylog multiple times after the plugin was installed or was the move to 2.1.1 the first time since installing the plugin? In other words, is it possible that a second restart was what made this start working for both of us? From my perspective, I can see no other reason.

**Comment by 123dev** [comment]
@brucegivens 
I don't think it has to do with the restart count, the version 2.1 of Graylog, we had restarted so many times and never saw the settings, version 2.1.1 we saw it on first start.
One thing I noticed that could be related, though I'm not certain is that , we typically load Graylog in a browser that is already at a specific page (example, search page, or a stream or overview ...)
Graylog on first load of /gettingstarted page, seems to be doing something different, and reports something along the lines of "we're preparing graylog .." (can't recall the exact wording)

Is it possible that configuration changes (when upgraded) are done only on /gettingstarted page.
What also supports this theory is that, the graylog reported version would reflect the older version, unless /gettingstarted is visited.

Just a theory.
